dll/win32/browseui/lang/fr-FR.rc

```diff
@@ -354,3 +354,9 @@ BEGIN
     IDS_COL_LOCATION "Dans le dossier"
     IDS_COL_RELEVANCE "Pertinence"
 END
+
+STRINGTABLE
+BEGIN
+    IDS_PARSE_ADDR_ERR_TITLE "File Explorer"
```

Comment:
```suggestion
    IDS_PARSE_ADDR_ERR_TITLE "Explorateur de fichiers"
```

Reply:
Note also (in all the translations) that we could replace "File Explorer" by "ReactOS Explorer" since this is denomination can be seen around:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=%28%5Cs%29explorer&sr=1

Reply:
Would you like me to also replace the french version with "ReactOS Explorer"?  Or keep the translated "Explorateur de fichiers"?

Reply:
It all depends on what you decide to do for the msgbox error title :)
If you decide to use "ReactOS Explorer", then yes for french it would be "Explorateur ReactOS".

Reply:
Went for ReactOS Explorer in the new commit.